app/views/employment/EmployerInformationView.scala.html

```diff
@@ -0,0 +1,108 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.templates.Layout
+@import views.html.templates.helpers.Button
+@import views.html.templates.helpers.Heading
+@import utils.ViewUtils.summaryListRow
+@import controllers.employment.routes._
+
+@this(
+        layout: Layout,
+        formWithCSRF: FormWithCSRF,
+        govukSummaryList: GovukSummaryList,
+        govukInsetText: GovukInsetText,
+        button: Button,
+        heading: Heading,
+        govUkButton: GovukButton
+)
+
+@(employerNameData: String, empId:String, benefitsDefined:Boolean, taxYear: Int, isInYear:Boolean)(implicit user: User[AnyContent], messages: Messages, appConfig: AppConfig)
+
+@titleForUse = @{messages("common.employerInformation")}
+
+@layout(pageTitle = titleForUse, taxYear = Some(taxYear), isAgent = user.isAgent) {
+
+    @heading(employerNameData, Some(messages("payeEmployment.caption", (taxYear - 1).toString, taxYear.toString)), "govuk-!-margin-bottom-2")
+
+    @if(isInYear) {
+        @govukInsetText(InsetText(
+            content = Text(messages(s"employment.employmentInfo.p1.${if(user.isAgent) "agent" else "individual"}", taxYear.toString))
+        ))
+    }
+
+    @{
+        def toSummaryListRow(key: String, value: String, link: Option[Call]): SummaryListRow = {
+            summaryListRow(
+                if(link.isDefined){
+                    HtmlContent(s"""<a class="govuk-link" id="${key.toLowerCase.replace(' ', '-')}_link" href="${link.get}">$key</a>""")
+                    } else {
+                    HtmlContent(key)
+                },
+                HtmlContent(value),
+                keyClasses = "govuk-!-width-two-thirds",
+                valueClasses = "govuk-!-width-one-third right-align",
+                actions = Seq()
+            )
+        }
+
+        val benefitsRowInYear: Seq[SummaryListRow] = {
+            if (benefitsDefined){
+                Seq(toSummaryListRow(messages("common.employmentBenefits"),messages("common.status.updated"), Some(CheckYourBenefitsController.show(taxYear, empId))))
+            } else {
+                Seq(toSummaryListRow(messages("common.employmentBenefits"),messages("common.status.cannotUpdate"), None))
+            }
+        }
+
+        val benefitsRowEOY: Seq[SummaryListRow] = {
+            if (benefitsDefined){
+                Seq(toSummaryListRow(messages("common.employmentBenefits"),messages("common.status.updated"), Some(CheckYourBenefitsController.show(taxYear, empId))))
+            } else {
+                Seq(toSummaryListRow(messages("common.employmentBenefits"),messages("common.status.notStarted"), Some(CheckYourBenefitsController.show(taxYear, empId))))
+            }
+        }
+
+        if(isInYear) {
+            govukSummaryList(
+                SummaryList(
```

Comment:
I am not sure but you might need to add some classes to the SummaryList. The story says you should use summary list as is in the prototype (which you do), but you do not use the following classes: govuk-summary-list__two-cells govuk-!-margin-bottom-9. I think because of that, the button is a bit too close to the table above.

If content design is happy with that, it is ok, otherwise you should probably have the following classes: 
govuk-summary-list govuk-summary-list__two-cells govuk-!-margin-bottom-9

Reply:
I forgot about that. I'll make those changes

Reply:
done. I added a margin-top to the button